tests/eos-test-utils.c

```diff
@@ -351,6 +351,140 @@ prepare_sysroot_contents (GFile *repo,
 }
 
 static gboolean
+generate_big_file_for_delta_update (GFile *all_commits_dir,
+                                    guint commit_no,
```

Comment:
Nitpick: s/`commit_no`/`commit_number`/ (abbreviation here is pointless and confusing).